src/index.js

```diff
@@ -10,60 +10,41 @@ const { connectDB, db } = require('./database/db');
 dotenv.config();
 
 // Create a new client instance
-const client = new Client({
-  intents: [
-    GatewayIntentBits.Guilds,
-    GatewayIntentBits.GuildMessages,
-    GatewayIntentBits.MessageContent,
-  ]
-});
+const client = new Client({ intents: [GatewayIntentBits.Guilds] });
 
-// Initialize commands collection
+// Load commands and events
 client.commands = new Collection();
+loadCommands(client);
+loadEvents(client);
```

Comment:
You cannot load commands and events before your client.login, this should be placed after line 38 client.login